examples/sw/benchmarks/README.md

```diff
@@ -62,6 +62,32 @@ seedcrc          : 0xe9f5
 Correct operation validated. See README.md for run and reporting rules.
 ```
 
+### Choice of ISA string
+
+Different ISAs (to choose different RISC-V ISA extensions) can be selected by
+passing the desired ISA string into `RV_ISA` when invoking make.
+
+```
+make -C ./examples/sw/bencharmsk/coremark clean
+make -C ./examples/sw/benchmarks/coremark RV_ISA=rv32imc
+```
+
+This will build CoreMark using the 'C' extension (compressed instructions).
+
+If CoreMark is already built a `clean` must be done before building
+for a new ISA string to ensure a full rebuild.
+
+The following ISA strings give the best performance for the Ibex configurations
+listed in the README:
```

Comment:
But poor performance for other configurations? :-)

Maybe something like "The following table shows which ISA string gives the best performance with each Ibex configuration listed in the README"? I'm not sure though - that still sounds slightly ambiguous (it implies that there's an ISA string to rule them all!). Any other ideas?

Reply:
I don't think there's much difference in meaning between the two sentences. Ultimately I'm just trying to simply communicate how someone can reproduce our numbers, I don't think we need to worry too much about the subtleties.